### PR TITLE
Localize outdated page

### PR DESCRIPTION
**Pull request by joesis**
For https://github.com/getlantern/lantern-internal/issues/691

### CODE REVIEW

**Comment by joesis** [comment]
This PR did the following:
1. Changed the code to build localized outdated.html for supported languages, so beta.getlantern.org.s3-website-us-east-1.amazonaws.com/outdated.html redirects to beta.getlantern.org.s3-website-us-east-1.amazonaws.com/fa_IR/outdated.html when browser language is Iran.
1. Removed obsolete `getlantern.org` directory.
1. Updated translations for outdated.html.

File: src/ch/index.html [hunk 1]
```diff
@@ -267,7 +267,7 @@ <h3>Easy</h3>
   </div>
 </footer>
    
-    <script src="../static/js/vendor/jquery-1.10.1.js"></script>
+    <script src="../static/js/vendor/jquery-1.10.1.min.js"></script>
```
Comment (joesis) [inline src/ch/index.html]: It also replaced js links to minified version.

File: src/outdated/index.html [hunk 2]
```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
```
Comment (joesis) [inline src/outdated/index.html]: The file is moved from `getlantern.org/pages/outdated.html`

**Comment by joesis** [comment]
@oxtoacart do you have time to have a look at this one, which is required by https://github.com/getlantern/lantern_aws/pull/855?

**Comment by oxtoacart** [comment]
@joesis I configured my Firefox browser to use fa_ir as the locale and then I tried hitting http://beta.getlantern.org.s3-website-us-east-1.amazonaws.com/outdated.html, and I always get redirected to http://beta.getlantern.org.s3-website-us-east-1.amazonaws.com/en_US/outdated.html.

**Comment by joesis** [comment]
Wow thanks @oxtoacart ! It's a bug affecting language detection on the home page too. I wonder how it passed my manual test in the first place. Fixed in https://github.com/getlantern/www/pull/68/commits/ea8728dba57af36dfc0eb84d6351aaad46420e61

**Comment by oxtoacart** [comment]
Okay, that fix worked for me, merging!

**Comment by joesis** [comment]
Thanks @oxtoacart! Deployed.

**Pull request merged**